rclcpp/include/rclcpp/timer.hpp

```diff
@@ -53,12 +53,17 @@ class TimerBase
    * \param clock A clock to use for time and sleeping
    * \param period The interval at which the timer fires
    * \param context node context
+   * \param autostart timer state on initialization
+   *
+   * In order to activate a timer that is not started on initialization,
+   * user should call the reset() method.
```

Comment:
@fujitatomoya this should address the other comment, could you take a look and let me know?